### DISCUSSION TITLE
Fans +Power G14 GA403UI

### DISCUSSION THREAD

**Discussion by MxEnJp** [seerge/g-helper#3378] [Q&A]
What would be the best settings for this model?. I feel that my laptop is overheating somewhat when running for example games ( I'd say I've seen cpu hit 94~c and not sure if that is normal)and not sure if something can be done. Would love to see examples of what is being ran normally for this model as far as settings:)

**Comment by seerge** [comment]
@MxEnJp hello,

Generally speaking default profiles (baked in BIOS) should be already good, as they are tuned for your specific device by the manufacturer.

If you want to tackle heat a bit, you can try to limit your CPU power in addition to everything. Modern Intel or AMD CPUs are quite efficient, and usually for most of the graphics-intensive games setting CPU power limit to something like 30-40W is more than enough.

For battery life just make sure you use Silent and Eco (or Optimized) and set Battery Saver mode in windows tray menu.
